Pinta.Tools/CoreToolsExtension.cs

```diff
@@ -66,6 +66,8 @@ public void Initialize ()
 			PintaCore.Tools.AddTool (new RoundedRectangleTool ());
 			PintaCore.Tools.AddTool (new EllipseTool ());
 			PintaCore.Tools.AddTool (new FreeformShapeTool ());
+			
+			PintaCore.Tools.CreateGoupedTools ();
```

Comment:
Typo in the function name (missing `r` for `Gouped`)

Reply:
It is not relevant any more due to code changes.